temoa/components/costs.py

```
@@ -715,3 +715,102 @@ def param_loan_annualize_rule(
+
+def poll_loan_costs(
+    loan_life: float,
+    loan_annualize: float,
+    capacity: float,
+    invest_cost: float,
+    process_life: int,
+    p_0: int,
+    p_e: int,
+    global_discount_rate: float,
+    vintage: int,
+    **kwargs: object,
```

Ruff (0.16.0)

[warning] Unused function argument: `kwargs`

(ARG001)